### ISSUE TITLE
Width of currency buttons on giving page inconsistent

### ISSUE THREAD

**Issue by rugk** [liberapay/liberapay.com#1025]
On the giving page, it looks like this:

![grafik](https://user-images.githubusercontent.com/11966684/37768755-02d0a1f4-2dcf-11e8-9f70-efc53ada33bb.png)

I think the width of them depends on the width of the name of the team/account.

**Comment by Changaco** [comment]
It's a Firefox bug, a mysterious extra space that shouldn't be there and doesn't exist in the DOM.

**Comment by rugk** [comment]
Okay, so did you report that?

I looked into the console, and I can only see some strange minor differences of the buttons (one seems to have some margin-left or so).

**Comment by Changaco** [comment]
No, I didn't report this issue to Mozilla. Feel free to do it if you want.

**Comment by rugk** [comment]
If you can explain what the underlying issue is… What exactly triggers the additional width on some buttons compared to others?

**Comment by Changaco** [comment]
The bug is in how Firefox renders `<select>` elements. The anomalous space is not from the DOM nor created by CSS.

**Comment by rugk** [comment]
Okay, had a look at it in the browser inspector… There is actually no select element for the small items.

See this gif:
![currency](https://user-images.githubusercontent.com/11966684/41412418-39a7757e-6fe0-11e8-8541-2e96905b98ae.gif)

So the select element adds some extra space, okay here is a bug report I could made. However, please add details or a reduced test case if you can:
https://bugzilla.mozilla.org/show_bug.cgi?id=1468724

So first thing I found out: The `options` element has some padding by default in FF. Removing that helps a bit, but not very much. It's number 3 in this pic (1 is the original):
![sizescompared](https://user-images.githubusercontent.com/11966684/41414069-48ea9f7a-6fe5-11e8-8383-9796518d7463.png)

(for more details see bugzilla)

Also it is really the content as it seems:
![selectsizecompareddiv](https://user-images.githubusercontent.com/11966684/41414096-5e2cf31a-6fe5-11e8-965e-f0e2df7960cc.png)

/cc @eldelacajita
